### PR TITLE
fozzie-components@v2.25.0 – Documentation for font optimisation and subsetting

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Documentation for font optimisation and subsetting.

### CODE REVIEW

File: stories/typography.stories.mdx [hunk 1]
```diff
@@ -71,3 +71,76 @@ This lazily loads in the `JustEatBasis-Bold` font to take over from the Faux Bol
...
+
+## Font Optimisation (subsetting)
+
+We currently use the JustEatBasis font on Just Eat platforms; a custom typeface created for use on our products.
+
+The base fontset comes with a huge number of glyphs and layout features that aren't required for use on the web. Therefore we optimise this base font using a process called font-subsetting. Subsetting is the process of removing parts of the base fontset, to leave only the glyphs and features that you require.
+
+For reference, the base Just Eat Regular font has the following properties:
+
+![./public/justeatbasis-base.png](./public/justeatbasis-base.png)
```
Comment (DamianMullins) [inline stories/typography.stories.mdx:84]: Do you need to add this image?

Comment (ashleynolan) [reply stories/typography.stories.mdx:84]: Good spot – forgot to add that in 🤦

**Review by DamianMullins — DISMISSED** [review]

**Review by DamianMullins — APPROVED** [review]

**Review by ross-gratton — COMMENTED** [review]
LGTM - Managed to install and subset the font. Might be worth mentioning that this install line `pip install fonttools brotli zopfli` needs to be run, because brotli and zopfli are needed as well as fonttools.
If people follow the blog they'll get that, if they shortcut it they might miss that point.

**Comment by ashleynolan** [comment]
> LGTM - Managed to install and subset the font. Might be worth mentioning that this install line `pip install fonttools brotli zopfli` needs to be run, because brotli and zopfli are needed as well as fonttools.
> If people follow the blog they'll get that, if they shortcut it they might miss that point.

Cool - I’ll check what steps are needed on windows too, and then add in some specifics about the python stuff if that sounds ok @ross-gratton ?

**Pull request merged**